src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RequestForReconsiderationCallbackHandler.java

```diff
@@ -71,7 +76,8 @@ protected Map<String, Callback> callbacks() {
 
     private CallbackResponse validateRequestEligibilityAndGetPartyDetails(CallbackParams callbackParams) {
         List<String> errors = new ArrayList<>();
-        if (callbackParams.getCaseData().getTotalClaimAmount().compareTo(BigDecimal.valueOf(1000)) > 0) {
+        if (!featureToggleService.isCaseProgressionEnabled()
+            && callbackParams.getCaseData().getTotalClaimAmount().compareTo(BigDecimal.valueOf(1000)) > 0) {
```

Comment:
@miguelMolina3691 Could you delete this bit: ```
!featureToggleService.isCaseProgressionEnabled()
            && 
```

Reply:
Removed